src/lsm/table.zig

```diff
@@ -432,18 +398,56 @@ pub fn TableType(
                     .op = options.address,
                     .timestamp = options.snapshot_min,
                     .request = builder.value_count,
-                    .size = block_size - @as(u32, @intCast(values_padding.len + block_padding.len)),
+                    .size = @sizeOf(vsr.Header) + builder.value_count * @sizeOf(Value),
```

Comment:
Would be good to assert these two calculations against each other, that they're the same.

Reply:
The

```zig
                    assert(block_size - header.size ==
                        (data.value_count_max - values.len) * @sizeOf(Value) + data.padding_size);
```

after we checksum is exactly that.

Reply:
You got this.